src/Adapter/Module/Tab/ModuleTabManagementSubscriber.php

```diff
@@ -58,8 +58,6 @@ public static function getSubscribedEvents()
         return [
             ModuleManagementEvent::INSTALL => 'onModuleInstall',
             ModuleManagementEvent::UNINSTALL => 'onModuleUninstall',
-            ModuleManagementEvent::ENABLE => 'onModuleEnable',
```

Comment:
Hello @clotairer @bogdan202 and thank you for the PR and the bug report. I understand the logic you added in Tab.php but I don't understand why the changes in ModuleTabManagementSubscriber.php are needed.

I'm also afraid the changes in ModuleTabManagementSubscriber will have other consequences and might change some other behaviors 🤔

Reply:
Hello @matks ,
In my opinion, these modifications don't have a significant impact on other behaviors since they modify only one subscriber, and other subscribers that handle enable/disable module events keep their behavior.
This subscriber was modified because, handling the enable/disable module event, it changes  the property 'enbaled' of the admin tab for all shop contexts